### DISCUSSION TITLE
ExporterOptions extends java.io.Serializable

### DISCUSSION THREAD

**Discussion by adoalonso** [primefaces/community#1407] [PrimeFaces] [answered]
Surprisingly the interface ExporterOptions does not extends java.io.Serializable, we can use this interface for excel or pdf exports, but in the view object it will warn that the attribute doesn't extends Serializable

**Comment by melloware** [comment]
I think it should. Please open a GitHub Issue ticket.

**Reply by melloware** [reply, marked as answer]
Issue opened: https://github.com/primefaces/primefaces/issues/11619

Fixed for 13.0.8
